lib/debuglet.js

```diff
@@ -215,7 +217,21 @@ Debuglet.prototype.scheduleBreakpointFetch_ = function(seconds) {
 
         default:
           that.logger_.info('\t' + response.statusCode + ' completed.');
-          that.updateActiveBreakpoints_(body.breakpoints || []);
+          var bps = (body.breakpoints || []).filter(function(bp) {
+            if (bp.action !== 'CAPTURE') {
+              that.logger_.warn('Found breakpoint with invalid action:', bp.action);
+              bp.status = new StatusMessage(StatusMessage.BREAKPOINT_ACTION,
+                BREAKPOINT_ACTION_MESSAGE, true);
+              that.debugletApi_.updateBreakpoint(bp, function(err/*, body*/) {
```

Comment:
Would it be possible to use `completeBreakpoint_` instead? It would be nice if that was the only place in this file that called `debugletApi_.updateBreakpoint`. Do this only if doesn't require bending backwards.